config.json.example

```diff
@@ -11,7 +11,7 @@
 		"mlb_news": true,
 		"countdowns": true,
 		"date": true,
-		"date_format": "%A, %B %-d"
+		"date_format": "%A, %B %d"
```

Comment:
What is the significance of this change?

Reply:
This was to fix a issue that I now can't reproduce. I'll revert it.